components/feature/media/src/main/java/mozilla/components/feature/media/service/MediaSessionServiceDelegate.kt

```diff
@@ -181,6 +171,7 @@ internal class MediaSessionServiceDelegate(
 
     @VisibleForTesting
     internal fun shutdown() {
+        service.stopForeground(true)
```

Comment:
What's the reason for stopping the foreground state here if we stop the service anyways? Can we add a comment explaining this?

Reply:
Sure, this is to remove the notification as well as letting the OS know that this service is not a foreground service anymore.  I'll add the comments.

Reply:
Doesn't the following `stopSelf()` accomplish this already? 🤔 🤷‍♂️

Reply:
Hmm you might be right.  Let me look into this.